website/content/docs/secrets/aws.mdx

```diff
@@ -118,6 +118,36 @@ management tool.
     EOF
     ```
 
+    Finally, the `policy_document` argument supports the same
+    [identity templating](https://www.vaultproject.io/docs/concepts/policies#templated-policies)
+    as for Vault policies. This enables advanced use cases such as restricting resources
+    based on claims provided via JWT authentication. For example with Gitlab:
+
+    ```text
+    $ vault write aws/roles/my-other-role \
+        policy_arns=arn:aws:iam::aws:policy/AmazonEC2ReadOnlyAccess,arn:aws:iam::aws:policy/IAMReadOnlyAccess \
+        iam_groups=group1,group2 \
+        credential_type=iam_user \
+        policy_document_template=true \
+        policy_document=-<<EOF
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Action": "ec2:*",
+          "Resource": "*",
+          "Condition": {
+            "StringEquals": {"aws:ResourceTag/GitlabProject": "{{identity.entity.metadata.gitlab_project}}"}
```

Comment:
I think the `identity.entity` is only available as `identity.entity.aliases.${accessor}.metadata.project_path` whereas the `$accessor` is `auth_jwt_106145ba` from:

```bash
$ vault auth list
Path      Type     Accessor               Description
----      ----     --------               -----------
jwt/      jwt      auth_jwt_106145ba      n/a
token/    token    auth_token_c6bf282d    token based credentials
```

I think it's important to highlight this, took me way to long to understand that you really have to specify that properly and can't just say `entity.metadata`.

Reply:
Indeed ! I've updated the tests and docs to reflect this.